public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -99,6 +99,9 @@ const Policy = (props: PolicyComponentProps) => {
   const matchers = normalizeMatchers(currentRoute);
   const hasMatchers = Boolean(matchers && matchers.length);
   const hasFocus = routesMatchingFilters.some((route) => route.id === currentRoute.id);
+  const childrenNodesIds = currentRoute.routes?.map((route) => route.id) ?? [];
+  const someChildrenMatchSearch =
```

Comment:
Since this is a draft; I think we should not highlight parent policies with matching children in the same way we do the _actual_ matches. The screenshot below is I think a good indicator for why that would be confusing :)

The parent policy isn't using the `grafana-default-email` contact point but is still highlighted, I find that a bit confusing :D

<img width="1075" alt="image" src="https://github.com/grafana/grafana/assets/868844/12d0d208-c344-4c0e-8f9c-52b405365fd5">

Reply:
maybe we could use another color for the parent?

Reply:
Let's table this for now and focus on the pagination and indicators for how many matchers are found :)